server/controllers/userController.js

```diff
@@ -38,7 +38,9 @@ async function verifyUser(req, res, next) {
           res.locals.error = 'Incorrect Password!';
           return next();
         } else {
-          res.locals.userId = existinguser.id;
+          // eslint-disable-next-line no-underscore-dangle
+          res.locals.userId = existinguser._id;
```

Comment:
i believe if you jsut do existinguser.id it will work, and you can get rid of the eslint ignore